lib/bootstrap/casper.js

```diff
@@ -56,11 +56,12 @@ var methodsToProvide = {
     withFrame: ['frameInfo', function then () {}],
     withPopup: ['popupInfo', function then () {}]
 };
+methodsToProvide.thenEvaluate.allowMore = true;
```

Comment:
I wonder if we could instead append `...` to the parameter name of any variadic arguments:

```javascript
thenEvaluate: [function fn () {}, 'replacements...']
```